### PR TITLE
OCPBUGS-2281: Re-enable serverless e2e tests

### PR DESCRIPTION
**Pull request by jerolimov**
**Fixes:**
https://issues.redhat.com/browse/OCPBUGS-2281

Reverts serverless part of #12114

### CODE REVIEW

**Comment by jerolimov** [comment]
> Can we revert this change as well https://github.com/openshift/console/blob/master/frontend/packages/dev-console/integration-tests/support/pages/functions/knativeSubscriptions.ts#L33-L39
> 
> this was done when the operator was not fully functional

@invincibleJai reverted this change as well, PTAL

**Comment by invincibleJai** [comment]
still looks like there is a failure with the serverless operator installation, https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_console/12243/pull-ci-openshift-console-master-e2e-gcp-console/1595690150958993408



```
"before all" hook for "Create knative workload using Container image with extrenal registry on Add page: KN-05-TC05 (example #1)": Perform actions on knative service and revision "before all" hook for "Create knative workload using Container image with extrenal registry on Add page: KN-05-TC05 (example #1)" 
....

Because this error occurred during a `before all` hook we are skipping all of the remaining tests. AssertionError AssertionError: Timed out retrying after 150000ms: Expected to find content: 'DependenciesInstalled, InstallSucceeded, VersionMigrationEligible' but never did.
```

File: frontend/packages/dev-console/integration-tests/support/pages/functions/knativeSubscriptions.ts [hunk 1]
```diff
@@ -30,12 +30,10 @@ export const createKnativeServing = () => {
...
-      // Enable it once the issue is fixed in Serverless operator
-      // cy.contains(
-      //   'DependenciesInstalled, DeploymentsAvailable, InstallSucceeded, Ready, VersionMigrationEligible',
-      //   { timeout: 150000 },
-      // ).should('be.visible');
+      cy.contains(
+        'DependenciesInstalled, DeploymentsAvailable, InstallSucceeded, Ready, VersionMigrationEligible',
+        { timeout: 150000 },
+      ).should('be.visible');
       cy.contains('DependenciesInstalled, InstallSucceeded, VersionMigrationEligible', {
```
Comment (invincibleJai) [inline frontend/packages/dev-console/integration-tests/support/pages/functions/knativeSubscriptions.ts:37]: we don't need this now as above you added 

```
cy.contains(
        'DependenciesInstalled, DeploymentsAvailable, InstallSucceeded, Ready, VersionMigrationEligible',
        { timeout: 150000 },
      ).should('be.visible');
```

File: frontend/packages/dev-console/integration-tests/support/pages/functions/knativeSubscriptions.ts [hunk 2]
```diff
@@ -30,12 +30,10 @@ export const createKnativeServing = () => {
...
         timeout: 150000,
       }).should('be.visible');
```
Comment (invincibleJai) [inline frontend/packages/dev-console/integration-tests/support/pages/functions/knativeSubscriptions.ts:39]: we don't need this now as above you added

```
cy.contains(
        'DependenciesInstalled, DeploymentsAvailable, InstallSucceeded, Ready, VersionMigrationEligible',
        { timeout: 150000 },
      ).should('be.visible');
```

File: frontend/packages/dev-console/integration-tests/support/pages/functions/knativeSubscriptions.ts [hunk 3]
```diff
@@ -30,12 +30,10 @@ export const createKnativeServing = () => {
...
-      // Currently disabling the check for ready and DeploymentsAvailable status due to https://issues.redhat.com/browse/SRVKS-953
-      // Enable it once the issue is fixed in Serverless operator
-      // cy.contains(
-      //   'DependenciesInstalled, DeploymentsAvailable, InstallSucceeded, Ready, VersionMigrationEligible',
-      //   { timeout: 150000 },
-      // ).should('be.visible');
+      cy.contains(
+        'DependenciesInstalled, DeploymentsAvailable, InstallSucceeded, Ready, VersionMigrationEligible',
+        { timeout: 150000 },
+      ).should('be.visible');
```
Comment (invincibleJai) [inline frontend/packages/dev-console/integration-tests/support/pages/functions/knativeSubscriptions.ts:36]: as we have this now, can remove below line 37 to 39

```
      cy.contains('DependenciesInstalled, InstallSucceeded, VersionMigrationEligible', {
        timeout: 150000,
      }).should('be.visible');
```

**Pull request merged**